data/registerPage.ts

```diff
@@ -0,0 +1,49 @@
+/* Assign to the Header Details */
```

Comment:
You don't have to export each and every variable like this instead try a structure like below.

export const loginPageData = {
loginPageTitle: "",
loginPageDescription: ""
loginButtonCaption:
loginEmailtxtCaption:
};